src/bin/elementary/run.c

```diff
@@ -21,6 +21,8 @@
 # include <direct.h> /* getcwd */
 #endif
 
+#include "Eina.h"
```

Comment:
When including other libraries (and not the headers defined for the current..."target" (?)), use `<>`:

```suggestion
#include <Eina.h>
```